src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -150,6 +151,7 @@ def new_normalize_text(s):
     def show_help(self, cli_name, nouns, parser, is_group):
         self.update_loaders_with_help_file_contents(nouns)
         super(AzCliHelp, self).show_help(cli_name, nouns, parser, is_group)
+        print(SURVEY_PROMPT)
```

Comment:
each time when typing "-h", print this prompt?

Reply:
yes, that's by design.

Reply:
Will users be bothered? Reduce frequency?

Reply:
Originally, we thought of providing an option to turn the survey prompt off. But since users can just ignore it, let's just leave it there and see how it goes. We can improve this when we have more feedback.

Reply:
Hi there!

Please consider this as feedback that this is kind of annoying :) Especially since that's a coloured prompt (whereas the actual help text is not).

I've filled in the survey… but I could only select "neither good nor bad" for all entries, as I've _just started using the CLI_ (which is why I'm looking at the help).

Sorry for intruding on your PR, I'll go back to being quiet now, thanks!

Reply:
Thanks for the feedback. We'll look at then.